draft-homburg-deleg-incremental-deleg.md

```diff
@@ -0,0 +1,607 @@
+For another example, if the triggering query is `www.faculty.university.example.` and the target zone `example.` then the incremental deleg name is `university._deleg.example.`
+
+Normal DNAME, CNAME and SVCB in AliasMode processing should happen as before, though note that when following an SVCB RR in AliasMode, the target name MUST have the `_dns` label prepended to the TargetName in the SVCB RR.
+The eventual incremental deleg query response, after following all redirections caused by DNAME, CNAME and AliasMode SVCB RRs, has three possible outcomes:
+
+1. An SVCB RRset in ServiceMode is returned in the response's answer section containing the delegation for the subzone.
+
+   The SVCB RRs in the RRset MUST be used to follow the referral.
+   The TargetName data field in the SVCB RRs in the RRset MUST be used as the names for the name servers to contact for the subzone, and the ipv4hint and ipv6hint parameters MUST be used as the IP addresses for the TargetName in the same SVCB RR.
+
+   The NS RRset and glue, in the response of the legacy query that was sent in parallel to the incremental deleg query, MUST NOT be used, but the signed DS record (or NSEC(3) records indicating that there was no DS) MUST be used in linking the DNSSEC authentication chain as which would conventionally be done with DNSSEC as well.
+
+2. The incremental deleg query name does not exist (NXDOMAIN).
+
+   There is no incremental delegation for the subzone, and the referral response for the legacy delegation MUST be processed  as would be done with legacy DNS and DNSSEC processing.
+
+3. The incremental deleg query name does exist, but resulted in an NOERROR no answer responses (also known as a NODATA response).
+
+   If the legacy query, did result in a referral for the same number of labels as the subdomain that the incremental deleg query was for, then there was no incremental delegation for the subzone, and the referral response for the legacy delegation MUST be processed as would be done with legacy DNS and DNSSEC processing.
+
+   Otherwise, the subzone may be more than one label below the delegating zone.
+
+   If the response to the legacy query resulted in a referral, then a new incremental deleg query MUST be spawned, matching the zone cut of the legacy referral response.
+   For example if the triggering query is `www.university.ac.example.` and the target zone `example.`, and the legacy response contained an NS RRset for `university.ac.example.`, then the incremental deleg query name is `university.ac._deleg.example.`
+   The response to the new incremental deleg query MUST be processed as described above, as if it was the initial incremental deleg query.
+
+   If the legacy query was send minimised and needs a followup query, then parallel to that followup query a new incremental deleg query will be send, adding a single label to the previous incremental deleg query name.
```

Comment:
/s/was send/was sent/  /s/will be send/will be sent/   (both future and past tense is sent).